travis/script-syntax.py

```diff
@@ -19,14 +16,24 @@ def main():
         if '.git' in parts or 'depends' in parts:
             continue
         for filename in filenames:
-            if not filename.endswith(ext):
+            if not filename.endswith(args.ext):
                 continue
             full_path = os.path.join(cur, filename)
             try:
                 subprocess.check_output(cmd + [full_path])
             except subprocess.CalledProcessError:
                 err = True
+            except IOError:
+                # catch error if not in Travis and Lua/Ruby is not available
```

Comment:
How is this useful? The only time this script is run automatically is in Travis (or travis/all.py), so if someone runs this script manually to check lua or ruby scripts, it shouldn't fail silently because lua/ruby aren't available.